homework-1/BackAccount.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+
+namespace BankingApplication
+{
+    public class BankAccount
+    {
+        private int _balance;
+
+        public BankAccount()
+        {
+            _balance = 0;
+        }
+
+        public void Deposit(int amount)
+        {
+            _balance += amount;
+            Console.WriteLine($"Deposited ₹{amount}. New _balance: {_balance}");
```

Comment:
This console acts like a user response and if possible user response messages should not have the exact variable name. So don't use `_balance` anywhere in the response message. Sorry for nitpicking